{"uid":"ade4ae7bdb637f5d","name":"Get the transaction by invalid hash","fullName":"tests.TransactionTests.get_transaction_by_invalid_hash","historyId":"ce3e5bc9e083438de734c55ea015b7be","time":{"start":1684434605337,"stop":1684434607040,"duration":1703},"status":"failed","statusMessage":"Incorrect error message expected [Transaction not found] but found [null]","statusTrace":"java.lang.AssertionError: Incorrect error message expected [Transaction not found] but found [null]\n\tat org.testng.Assert.fail(Assert.java:97)\n\tat org.testng.Assert.failNotEquals(Assert.java:969)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:131)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:575)\n\tat microservices.common.steps.BaseSteps.then_verifyErrorResponse(BaseSteps.java:15)\n\tat tests.TransactionTests.get_transaction_by_invalid_hash(TransactionTests.java:28)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\tat java.lang.Thread.run(Thread.java:750)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setUp","time":{"start":1684434605335,"stop":1684434605336,"duration":1},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0}],"testStage":{"status":"failed","statusMessage":"Incorrect error message expected [Transaction not found] but found [null]","statusTrace":"java.lang.AssertionError: Incorrect error message expected [Transaction not found] but found [null]\n\tat org.testng.Assert.fail(Assert.java:97)\n\tat org.testng.Assert.failNotEquals(Assert.java:969)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:131)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:575)\n\tat microservices.common.steps.BaseSteps.then_verifyErrorResponse(BaseSteps.java:15)\n\tat tests.TransactionTests.get_transaction_by_invalid_hash(TransactionTests.java:28)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\tat java.lang.Thread.run(Thread.java:750)\n","steps":[{"name":"Get transaction by hash","time":{"start":1684434605337,"stop":1684434607035,"duration":1698},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"hash","value":"@#$%"}],"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true,"stepsCount":0},{"name":"Verify error response","time":{"start":1684434607035,"stop":1684434607040,"duration":5},"status":"failed","statusMessage":"Incorrect error message expected [Transaction not found] but found [null]","statusTrace":"java.lang.AssertionError: Incorrect error message expected [Transaction not found] but found [null]\n\tat org.testng.Assert.fail(Assert.java:97)\n\tat org.testng.Assert.failNotEquals(Assert.java:969)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:131)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:575)\n\tat microservices.common.steps.BaseSteps.then_verifyErrorResponse(BaseSteps.java:15)\n\tat tests.TransactionTests.get_transaction_by_invalid_hash(TransactionTests.java:28)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\tat java.lang.Thread.run(Thread.java:750)\n","steps":[],"attachments":[],"parameters":[{"name":"statusCode","value":"400"},{"name":"message","value":"Transaction not found"},{"name":"errorCode","value":"BC_404-TRANSACTION_NOT_FOUND"}],"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true,"stepsCount":0}],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true,"stepsCount":2},"afterStages":[{"name":"tearDown","time":{"start":1684434607042,"stop":1684434607043,"duration":1},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0}],"labels":[{"name":"package","value":"tests.TransactionTests"},{"name":"testClass","value":"tests.TransactionTests"},{"name":"testMethod","value":"get_transaction_by_invalid_hash"},{"name":"parentSuite","value":"Setel Regression Suite"},{"name":"suite","value":"Setel Admin test on chrome"},{"name":"subSuite","value":"tests.TransactionTests"},{"name":"host","value":"fv-az565-46"},{"name":"thread","value":"2371@fv-az565-46.TestNG-test=Setel Admin test on chrome-2(13)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"arg0","value":"@#$%"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":4},"items":[{"uid":"4328dd6113cabad6","reportUrl":"https://Sotatek-DavidNguyen4.github.io/cardano-api-tests/59//#testresult/4328dd6113cabad6","status":"failed","statusDetails":"Incorrect error message expected [Transaction not found] but found [null]","time":{"start":1684408366953,"stop":1684408368535,"duration":1582}},{"uid":"d57417ace44f4fa6","reportUrl":"https://Sotatek-DavidNguyen4.github.io/cardano-api-tests/58//#testresult/d57417ace44f4fa6","status":"failed","statusDetails":"Incorrect error message expected [Transaction not found] but found [null]","time":{"start":1684385008372,"stop":1684385010381,"duration":2009}},{"uid":"610fdceff0039b1f","reportUrl":"https://Sotatek-DavidNguyen4.github.io/cardano-api-tests/57//#testresult/610fdceff0039b1f","status":"failed","statusDetails":"Incorrect error message expected [Transaction not found] but found [null]","time":{"start":1684348180447,"stop":1684348182485,"duration":2038}}]},"tags":[]},"source":"ade4ae7bdb637f5d.json","parameterValues":["@#$%"]}